### DISCUSSION TITLE
does color theme allow rgba?

### DISCUSSION THREAD

**Discussion by LollipopFt** [helix-editor/helix#10456] [General]
so i was trying out making a color theme but realised that `#ffffff20` still output full white. is there any way i can make it have transparency? thanks!

**Comment by NNBnh** [comment]
Cool idea but nope, it's impossible due to limitation of the terminal, maybe this will be imperment to the GUI forntend.

Edit:

I remember saw this on [Kakoune's faces docs](https://github.com/mawww/kakoune/blob/master/doc/pages/faces.asciidoc):

> **rgba:RRGGBBAA**
>
> hexadecimal value if unspecified or empty, default is used. Alpha values are used to blend the face onto either its base or the existing color. For technical reasons alpha values must be >16 if specified.

So yeah... We could implement it like that.
